lib/fpm/package/gem.rb

```diff
@@ -91,21 +101,33 @@ def download(gem_name, gem_version=nil)
 
     logger.info("Trying to download", :gem => gem_name, :version => gem_version)
 
-    gem_fetch = [ "#{attributes[:gem_gem]}", "fetch", gem_name]
-
-    gem_fetch += ["--prerelease"] if attributes[:gem_prerelease?]
-    gem_fetch += ["--version", gem_version] if gem_version
-
     download_dir = build_path(gem_name)
     FileUtils.mkdir(download_dir) unless File.directory?(download_dir)
 
-    ::Dir.chdir(download_dir) do |dir|
-      logger.debug("Downloading in directory #{dir}")
-      safesystem(*gem_fetch)
+    if attributes[:gem_git_repo]
+      logger.debug("Git cloning in directory #{download_dir}")
+      g = Git.clone(attributes[:gem_git_repo],gem_name,:path => download_dir)
```

Comment:
@jordansissel I am not much of a rubyist. Would I just need to move the `require "git"` line to the top of this block for #1748 ?